client/src/components/layouts/Default.js

```diff
@@ -163,11 +159,12 @@ class DefaultLayout extends Component {
           <style>{fontawesome.dom.css()}</style>
         </Helmet>
         <Header
-          disableMenuButtonBehavior={disableMenuButtonBehavior}
           disableSettings={disableSettings}
-          mediaBreakpoint={mediaBreakpoint}
+          navigationMenu={navigationMenu}
         />
-        <div className={`default-layout ${landingPage ? 'landing-page' : ''}`}>
+        <div
+          className={`default-layout ${disableSettings ? 'landing-page' : ''}`}
```

Comment:
It seems strange. It may be better to replace the “disableSettings” prop with “landingPage”, since we only hide/disable elements on the landing page.

Reply:
Agreed.  We only need one of them and I'm fine with "landingPage" being the one we use.